app/models/policy.rb

```diff
@@ -81,6 +93,13 @@ def destroy_orphaned_business_objective
 
   private
 
+  def unassigned_minor_versions
+    stored_versions = profiles.pluck(:os_minor_version)
+    Host.os_minor_versions(hosts.pluck(:id)).reject do |version|
+      stored_versions.include?(version) # Ignore already stored minor versions
+    end
+  end
```

Comment:
This doesn't quite work as expected (string vs integer comparison is the culprit):

```rb
[304] pry(main)> Policy.first.send(:unassigned_minor_versions)
  Policy Load (1.3ms)  SELECT  "policies".* FROM "policies" ORDER BY "policies"."id" ASC LIMIT $1  [["LIMIT", 1]]
   (1.0ms)  SELECT "profiles"."os_minor_version" FROM "profiles" WHERE "profiles"."policy_id" = $1  [["policy_id", "e1e6e24e-46e2-4f9f-8986-50432875a6f5"]]
   (19.3ms)  SELECT "inventory"."hosts"."id" FROM "inventory"."hosts" INNER JOIN "policy_hosts" ON "inventory"."hosts"."id" = "policy_hosts"."host_id" WHERE "policy_hosts"."policy_id" = $1  [["policy_id", "e1e6e24e-46e2-4f9f-8986-50432875a6f5"]]
   (18.2ms)  SELECT DISTINCT system_profile->'operating_system'->'minor' FROM "inventory"."hosts" WHERE "inventory"."hosts"."id" = $1  [["id", "78a1ce1a-6173-484a-bcaa-9b10a46e651b"]]
=> [9]
[305] pry(main)> Policy.first.profiles.map { |p| [p.id, p.os_minor_version, p.ssg_version] }
  Policy Load (1.0ms)  SELECT  "policies".* FROM "policies" ORDER BY "policies"."id" ASC LIMIT $1  [["LIMIT", 1]]
  Profile Load (0.7ms)  SELECT "profiles".* FROM "profiles" WHERE "profiles"."policy_id" = $1  [["policy_id", "e1e6e24e-46e2-4f9f-8986-50432875a6f5"]]
  Xccdf::Benchmark Load (0.6ms)  SELECT  "benchmarks".* FROM "benchmarks" WHERE "benchmarks"."id" = $1 LIMIT $2  [["id", "52a35239-0b11-46c0-a372-c32f7d42dc8f"], ["LIMIT", 1]]
  Xccdf::Benchmark Load (0.6ms)  SELECT  "benchmarks".* FROM "benchmarks" WHERE "benchmarks"."id" = $1 LIMIT $2  [["id", "2e08ee4c-7114-4671-9227-fcc636df8653"], ["LIMIT", 1]]
=> [["303dce8f-053d-4d38-974d-e464c27ed0e0", "9", "0.1.52"], ["4948c203-345b-4501-9c1d-48752dd97611", "", "0.1.54"]]
```